internal/cmd/gocdk/init.go

```diff
@@ -81,6 +82,16 @@ func init_(ctx context.Context, pctx *processContext, args []string) error {
 	if err := materializeTemplateDir(projectDir, "init", tmplValues); err != nil {
 		return xerrors.Errorf("gocdk init: %w", err)
 	}
+	fmt.Fprintf(pctx.stdout, "Project created at %s with:\n", projectDir)
```

Comment:
WDYT about putting this content in `static`? It might be easier to edit that way.

Reply:
It's short enough and a single message that I don't think the separation buys much.